app/models/fabrique_evenement.rb

```diff
@@ -10,8 +10,11 @@ def initialize(parametres)
   def call
     partie = recupere_partie
     evenement = Evenement.new EvenementParams.from(parametres).merge(partie: partie)
-    evenement.save
-    PersisteMetriquesPartieJob.perform_later(partie) if evenement.fin_situation?
+    evenement.save do |saved|
+      next unless saved
+
+      PersisteMetriquesPartieJob.perform_later(partie) if evenement.fin_situation?
+    end
     evenement
```

Comment:
Je ne suis pas trop familier avec cette syntaxe, j'aurais plutot envie de l'écrire comme cela : 

```suggestion
    return unless evenement.save
    
    PersisteMetriquesPartieJob.perform_later(partie) if evenement.fin_situation?
    evenement
```